playground/playground.js

```diff
@@ -18,10 +18,24 @@ if (handleParam) {
   handle.value = handleParam;
 }
 
+function createBSkyIntent(msg) {
+  // If I'm on windows it should be <br>, if not it should be \n
+  const lineBreak = navigator.userAgent.toLowerCase().includes("windows")
+    ? "<br>"
+    : "\n";
```

Comment:
This removes the line breaks from mac also 🤔 

I think it depends on whether you're opening from browser or app? like my current implementation seems to work for me on browser but breaks when I share from mobile app



<img width="729" alt="image" src="https://github.com/user-attachments/assets/e93a25b1-1e4e-4de5-a80c-d4590e04408a">

Reply:
Hummm, let's keep track https://github.com/bluesky-social/social-app/issues/6133